packages/core/src/families/common.ts

```diff
@@ -5,7 +5,7 @@ export const schemaTransactionCommon = z.object({
   recipient: z.string(),
 });
 
-export const schemaFamilies = z.enum([
+export const FAMILIES = [
```

Comment:
I would suggest:
```suggestion
export const Families = [
```

cf. [this comment](https://github.com/LedgerHQ/live-app-sdk/blob/main/src/families/types.ts#L1-L7) I put back in the day

Reply:
We already export a type `Families` in `families/types.ts` and I wanted to avoid clashing with it

Reply:
Fine, we can keep this `FAMILIES` then I guess 😞